examples/gallery/lines/linestrings.py

```diff
@@ -0,0 +1,39 @@
+"""
+Plotting lines with LineString/MultiLineString geometry
+=======================================================
+
+The :meth:`pygmt.Figure.plot` method allows us to plot geographical data such
+as lines which are stored in a :class:`geopandas.GeoDataFrame` object. Use
+:func:`geopandas.read_file` to load data from any supported OGR format such as
+a shapefile (.shp), GeoJSON (.geojson), geopackage (.gpkg), etc. Then, pass the
+:class:`geopandas.GeoDataFrame` as an argument to the ``data`` parameter of
+:meth:`pygmt.Figure.plot`, and style the geometry using the ``pen`` parameter.
+"""
+
+# %%
+import geodatasets
+import geopandas as gpd
+import pygmt
+
+# Read shapefile data of large rivers in Europe using geopandas
+gdf = gpd.read_file(geodatasets.get_path("eea large_rivers"))
+
+# Convert object to EPSG 4326 coordinate system
+gdf = gdf.to_crs("EPSG:4326")
```

Comment:
Check the content of the data before plotting.
```suggestion
gdf = gdf.to_crs("EPSG:4326")
print(gdf)
```